src/Symfony/Component/Form/Extension/Core/DataTransformer/NumberToLocalizedStringTransformer.php

```diff
@@ -93,13 +93,14 @@ class NumberToLocalizedStringTransformer implements DataTransformerInterface
      */
     const ROUND_HALFDOWN = \NumberFormatter::ROUND_HALFDOWN;
 
+    // todo rename to scale in Symfony 3.0
```

Comment:
right, then only the comment needs an update, e.g `// @deprecated: will be replaced by a $scale private property in 3.0`